.github/ISSUE_TEMPLATE/4-release.yml

```diff
@@ -0,0 +1,123 @@
+name: 'WooCommerce: release checklist'
+description: Create a new issue to track the next WooCommerce core release progress.
+title: "WooCommerce RELEASE_VERSION: RC, final and dot-releases progress tracking"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This issue provides visibility on the progress of the release process of WooCommerce core RELEASE_VERSION and 
+        it centralizes all related discussions and documentation. The ultimate goal of this issue is to keep a reference to the steps,
+        resources, work, and conversations associated with this release so it can be helpful for future contributors handling the release of
+        a new WooCommerce core version.
+
+        -   WooCommerce core version to release: RELEASE_VERSION ([milestone](https://github.com/woocommerce/woocommerce/milestone/MILESTONE_ID))
+        -   Release lead: RELEASE_LEAD
+        -   RC release date: RELEASE_DATE
+        -   Final release date: RELEASE_DATE
+        -   WooCommerce core change log: [here](https://github.com/woocommerce/woocommerce/blob/trunk/changelog.txt)
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Resources
+
+        -   [Release calendar](https://developer.woocommerce.com/release-calendar/)
+        -   [Previous releases](https://developer.woocommerce.com/releases/)
+        -   Latest (version 9.9+) [release process changes](https://developer.woocommerce.com/2025/03/05/upcoming-changes-to-the-woocommerce-release-process/)
+        -   Internal detailed reference to the release process: Pf59Ax-7F-p2
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Checklist
+
+  - type: markdown
+    attributes:
+      value: |
+        ### RC release
```

Comment:
Again, there are 2 RC releases, so we might need 2 identical sections? Or maybe we can find a way to condense things.

Reply:
Will this work: 8091b6c547ce16b51648b49ff7f7a5d1b9e21b50 ?

Reply:
> Plus that means that we won't have checklists for each RC, right?

Yep

> I find 8091b6c547ce16b51648b49ff7f7a5d1b9e21b50 a bit confusing

Ok, I'll revert. Do we want to have a separate checklist for RC 1 and RC 2, then?